.github/workflows/main-push-autotag.yml

```diff
@@ -39,6 +39,7 @@ jobs:
       run: |
         echo "Attempting to auto-tag merge commit"
         new_tag=$(.github/auto-tag.sh "${PR_BODY}" "${PR_NUMBER}")
+        new_tag=$(echo "${new_tag}" | tail -n 1)
```

Comment:
I'm going to whine about the name :) `new_tag` here should probably be `script_output` or something :stuck_out_tongue_closed_eyes:

Reply:
Done!